src/Initializer/dg_setup.f90

```diff
@@ -1953,67 +855,12 @@ REAL    :: phi                                                            ! Valu
     DO iElem = 1,MESH%nElem
         l_initialLoading=0
         l_plasticParameters=0
         
-        IF(EQN%Plasticity.EQ.1 .AND. EQN%PlastMethod .EQ. 2) THEN
-          x = 0.; y = 0.; z = 0.;
-          eType = MESH%LocalElemType(iElem)
-          SELECT CASE(eType)
-          CASE(4)
-              DO iVert=1,MESH%nVertices_Tet
-                  x(iVert) = MESH%VRTX%xyNode(1,MESH%ELEM%Vertex(iVert,iElem))
-                  y(iVert) = MESH%VRTX%xyNode(2,MESH%ELEM%Vertex(iVert,iElem))
-                  z(iVert) = MESH%VRTX%xyNode(3,MESH%ELEM%Vertex(iVert,iElem))
-              ENDDO
-              ! Point to the corresponding:
-              ! Integration points
-              intGaussP     => DISC%Galerkin%intGaussP_Tet
-              intGaussW     => DISC%Galerkin%intGaussW_Tet
-              ! Basis func values
-              IntGPBaseFunc => DISC%Galerkin%IntGPBaseFunc_Tet(1:nDegFr,1:nIntGp,iPoly)
-              ! Mass matrix
-              MassMatrix    => DISC%Galerkin%MassMatrix_Tet(1:nDegFr,1:nDegFr,iPoly)
-          CASE(6)
-              DO iVert=1,MESH%nVertices_Hex
-                  x(iVert) = MESH%VRTX%xyNode(1,MESH%ELEM%Vertex(iVert,iElem))
-                  y(iVert) = MESH%VRTX%xyNode(2,MESH%ELEM%Vertex(iVert,iElem))
-                  z(iVert) = MESH%VRTX%xyNode(3,MESH%ELEM%Vertex(iVert,iElem))
-              ENDDO
-              ! Point to the corresponding:
-              ! Integration points
-              intGaussP     => DISC%Galerkin%intGaussP_Hex
-              intGaussW     => DISC%Galerkin%intGaussW_Hex
-              ! Basis func values
-              IntGPBaseFunc => DISC%Galerkin%IntGPBaseFunc_Hex(1:nDegFr,1:nIntGp,iPoly)
-              ! Mass matrix
-              MassMatrix    => DISC%Galerkin%MassMatrix_Hex(1:nDegFr,1:nDegFr,iPoly)
-          END SELECT
-
-          DO iIntGP = 1,nIntGP
-              IF(EQN%Plasticity.EQ.1 .AND. EQN%PlastMethod .EQ. 2) THEN !average approach for plasticity
-              ! L2 projection of initial stress loading for the plastic calculations onto the DOFs
-                iniGP_Plast(:) = EQN%IniStress(1:6,iElem)
-                DO iDegFr = 1, nDegFr
-                   phi = IntGPBaseFunc(iDegFr,iIntGP)
-                   l_initialLoading(iDegFr,1:6) = l_initialLoading(iDegFr,1:6) + IntGaussW(iIntGP)*iniGP_plast(:)*phi
-                ENDDO
-             ENDIF
-          ENDDO !iIntGP
-
-          DO iDegFr = 1, nDegFr
-            l_initialLoading(iDegFr, :) = l_initialLoading( iDegFr, : ) / massMatrix(iDegFr,iDegFr)
-          ENDDO
-
-          NULLIFY(intGaussP)
-          NULLIFY(intGaussW)
-          NULLIFY(IntGPBaseFunc)
-          NULLIFY(MassMatrix)
-        ENDIF
-
-        IF(EQN%Plasticity.EQ.1 .AND. EQN%PlastMethod .EQ. 0) THEN !high-order points approach
+        IF(EQN%Plasticity .EQ. 1) THEN !high-order points approach
```

Comment:
That confused me - isn't PlastMethod used anymore?

Reply:
That parameter refers to https://github.com/SeisSol/SeisSol/blob/master/src/Solver/plasticity.f90#L286
But that routine has never been ported to C++ and is not necessary, too.